diff --git a/dists/sailfish/org.scummvm.scummvm.desktop b/dists/sailfish/org.scummvm.scummvm.desktop
index 9d4f4f03..4f1a8eab 100644
--- a/dists/sailfish/org.scummvm.scummvm.desktop
+++ b/dists/sailfish/org.scummvm.scummvm.desktop
@@ -7,13 +7,18 @@ Comment[he]=פרשן למספר משחקי הרפתקאות
 Comment[de]=Interpreter für zahlreiche Abenteuerspiele und RPGs
 Comment[es]=Intérprete para varias aventuras gráficas
 Comment[ca]=Intèrpret per diverses aventures gràfiques
-Exec=/usr/bin/org.scummvm.scummvm
-Icon=org.scummvm.scummvm
+Exec=/usr/bin/org.scummvm.scummvm -initial-cfg=/etc/scummvm/scummvm.ini
+Icon=icon-launcher-scummvm
 Type=Application
 Categories=Game;AdventureGame;
 X-Nemo-Application-Type=no-invoker
 X-Desktop-File-Install-Version=0.26
 
+[X-Sailjail]
+Permissions=UserDirs;RemovableMedia;Internet;Audio
+OrganizationName=org.scummvm
+ApplicationName=scummvm
+
 [X-Application]
 Permissions=UserDirs;RemovableMedia;Internet;Audio
 OrganizationName=org.scummvm
