diff --git a/dists/sailfish/org.scummvm.scummvm.desktop b/dists/sailfish/org.scummvm.scummvm.desktop
index 9d4f4f03..cb5b69a6 100644
--- a/dists/sailfish/org.scummvm.scummvm.desktop
+++ b/dists/sailfish/org.scummvm.scummvm.desktop
@@ -7,7 +7,7 @@ Comment[he]=פרשן למספר משחקי הרפתקאות
 Comment[de]=Interpreter für zahlreiche Abenteuerspiele und RPGs
 Comment[es]=Intérprete para varias aventuras gráficas
 Comment[ca]=Intèrpret per diverses aventures gràfiques
-Exec=/usr/bin/org.scummvm.scummvm
+Exec=/usr/bin/org.scummvm.scummvm -initial-cfg=/etc/scummvm/scummvm.ini
 Icon=org.scummvm.scummvm
 Type=Application
 Categories=Game;AdventureGame;
